docker-compose.yml

```diff
@@ -7,17 +7,37 @@ services:
       - 9432:9432
     restart: always
 
+  gitcollector:
+    image: srcd/gitcollector:v0.0.1-beta.3
+    # wait for db
```

Comment:
Maybe dumb question, but which db? The `superset`'s postgres? Isn't the
```
depends_on:
  - postgres
```
enough?

Reply:
no. not enough. `depends_on` handles only container creation. But postgres needs time to init and start before becoming available. You can read more about it here: https://docs.docker.com/compose/startup-order/

Reply:
thanks for the link!